craco.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
I'm not really familiar with CRACO. Any reasons why you're using it over react-scripts?

Reply:
@xarrijorge CRACO is used to override default CRA settings without ejecting. As far as I can tell it isn't possible to use CRA with TailwindCSS without ejecting or using something like CRACO, because CRA doesn't support the version of PostCSS that Tailwind requires. 
See https://tailwindcss.com/docs/guides/create-react-app for details.